### DISCUSSION TITLE
Toggle campfire,fountain,etc

### DISCUSSION THREAD

**Discussion by zaneshaw** [Pryaxis/TShock#2012] [General]
What is the permission node for toggle on/off campfires and fountains?

**Comment by zaneshaw** [comment]
bump

**Comment by moisterrific** [comment]
Wdym permission node? You need permission to toggle campfires?

**Reply by zaneshaw** [reply]
Yes. I just found out you can toggle campfires and other outside of the spawn area (200 blocks?) and I already have spawn protection turned down and off.
I mean permission node by `tshock.permssion.x` but this might not be a permission thing. Superadmin can toggle near spawn though that's why I assumed it was.

**Reply by chris-der-kleine** [reply]
As it seems that didn't do the trick for me.  

I'm able to place a fountain, but when I try to interact with it, the server just does nothing

**Comment by zaneshaw** [comment]
I fixed my issue with this guide. You should try it out. You will also have to remove the thsock.ignore.ssc if you want ssc enabled on your server. #2065
